src/main/java/Duke.java

```diff
@@ -1,10 +1,84 @@
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+    public static void main(String[] args) throws DukeException {
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+        Task[] list = new Task[100];
+        int listcounter = 0;
+
+        while(true){
+            Scanner sc = new Scanner(System.in);
+            String line = sc.nextLine();
+            String[] action = line.split(" ", 2);
+
+            if(action[0].equals("bye")){
+                System.out.println("Bye. Hope to see you again soon!");
+                break;
+
+            } else if(action[0].equals("list")){
+                System.out.println("Here are the tasks in your list:");
+                for(int i=0;i<listcounter;i++){
+                    System.out.println((i + 1) + "." + list[i].toString());
+                }
+
+            } else if(action[0].equals("done")){
+                try {
+                    int a = Integer.parseInt(action[1]);
```

Comment:
Could maybe change the naming of the variable 'a' to give it a more meaningful naming